creator-node/src/utils.js

```diff
@@ -95,6 +96,49 @@ const ipfsSingleByteCat = (path, req, timeout = 1000) => new Promise(async (reso
   }
 })
 
+const ipfsCat = (path, req, timeout = 1000, length = null) => new Promise(async (resolve, reject) => {
+  const start = Date.now()
+  let ipfs = req.app.get('ipfsLatestAPI')
+
+  try {
+    let chunks = []
+    let options = {}
+    if (length) options.length = length
+    if (timeout) options.timeout = timeout
+    // ipfs.cat() returns an AsyncIterator<Buffer> and its results are iterated over in a for-loop
+    /* eslint-disable-next-line no-unused-vars */
+    for await (const chunk of ipfs.cat(path, options)) {
+      chunks.push(chunk)
+    }
+    req.logger.info(`ipfsCat - Retrieved ${path} in ${Date.now() - start}ms`)
+    resolve(Buffer.concat(chunks))
+  } catch (e) {
+    req.logger.error(`ipfsCat - Error: ${e}`)
+    reject(e)
+  }
+})
+
+const ipfsGet = (path, req, timeout = 1000) => new Promise(async (resolve, reject) => {
+  const start = Date.now()
+  let ipfs = req.app.get('ipfsLatestAPI')
+
+  try {
+    let chunks = []
+    let options = {}
+    if (timeout) options.timeout = timeout
+    // ipfs.get() returns an AsyncIterator<Buffer> and its results are iterated over in a for-loop
+    /* eslint-disable-next-line no-unused-vars */
+    for await (const chunk of ipfs.get(path, options)) {
+      chunks.push(chunk)
```

Comment:
per the example, it seems like this function is not constructed correctly - it should be appending chunk.content https://github.com/ipfs/js-ipfs/blob/master/docs/core-api/FILES.md#ipfsgetipfspath-options

Reply:
@SidSethi

Reply:
fixed